lib/hobbes/lang/preds/hasfield.C

```diff
@@ -11,7 +11,7 @@
 
 namespace hobbes {
 
-#define FIELD_VALUE_FN "fieldValue"
+static constexpr char const* FIELD_VALUE_FN = "fieldValue";
```

Comment:
?

Reply:
There is no clear benefits for this change in this case. Just in general, I prefer C++ style instead of C